keel-deliveryconfig-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/deliveryconfig/DeliveryConfig.kt

```diff
@@ -1,10 +1,21 @@
 package com.netflix.spinnaker.keel.api.deliveryconfig
 
+import com.fasterxml.jackson.annotation.JsonAnyGetter
+import com.fasterxml.jackson.annotation.JsonAnySetter
 import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
 
 @JsonInclude(NON_NULL)
 data class DeliveryConfig(
   val name: String,
-  val application: String
-)
+  val application: String,
+  val deliveryArtifacts: List<Map<String, Any>> = emptyList(),
+  val deliveryEnvironments: List<DeliveryEnvironment> = emptyList(),
+  @get:JsonAnyGetter val otherAttributes: MutableMap<String, Any> = mutableMapOf()
```

Comment:
this should be an immutable map, I'm not sure how I feel about json any setters and data classes yet (I could go the route of a constructor that takes Map<String, Any> but that feels similarly bad

Reply:
removing it entirely, it was mostly there for when I was expecting to pull this from front50